server/data/statuses.json

```diff
@@ -0,0 +1,43 @@
+[
+    {
+        "id": "NEW",
+        "name": "New"
+    },
+    {
+        "id": "ADDED_IN_LINKEDIN",
+        "name": "Added in LinkedIn"
```

Comment:
@evasilkovich I think Status should be abstract. "Added in LinkedIn" rely to SOURCE_ID 
Can we review this status ?